pywt/_cwt.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+
+from ._extensions._pywt import DiscreteContinuousWavelet, ContinuousWavelet, Wavelet, Modes, _check_dtype
+from ._extensions._cwt import (cwt_psi_single)
+from ._functions import integrate_wavelet, scale2frequency
+
+__all__ = ["cwt"]
+
+
+def cwt(data, scales, wavelet, sampling_period=1.):
+    """
+    cwt(data, scales, wavelet)
+
+    One dimensional Continuous Wavelet Transform.
+
+    Parameters
+    ----------
+    data : array_like
+        Input signal
+    scales : array_like
+        scales to use
+    wavelet : Wavelet object or name
+        Wavelet to use
+    sampling_period : float
+        Sampling period for frequencies output (optional)
+
+    Returns
+    -------
+    coefs : array_like
+        Continous wavelet transform of the input signal for the given scales and wavelet
+    frequencies : array_like
+        if the unit of sampling period are seconds and given, than frequencies are in hertz. Otherwise Sampling period of 1 is assumed.
```

Comment:
I just sent a PR to your master branch containing some PEP8 related style fixes I made while reviewing the code.  That was easier than line commenting individually.